src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -47,14 +47,31 @@ public CommandResult execute() {
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        Set<String> lowerCaseKeywords = convertSetOfStringsToLowerCase(keywords);
```

Comment:
Can this variable be final? Keeping variables final can help prevent the developer from accidentally reassigning it.